recipes/nyuki_recipe/meta.yaml

```diff
@@ -0,0 +1,71 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "nyuki" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source: 
+  url: https://github.com/00krishna-tools/nyuki/archive/v{{ version }}.tar.gz
+  sha256: 88e943eed59d5ca95d6716b22b47c71a7b7c9323a71348a4623e329bb66d7ae7
+
+build:
+  noarch: python
+  number: 0
+  skip: true  # [py<36]
+  entry_points:
+    - nyuki = nyuki.nyuki:nyuki
+#  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >3.5
+    - pip
+    - numpy >=1.18.5
+    - pandas
+    - rasterio
+    - click >=7
+    - tqdm
+    - geopandas
+    - Pillow
+    - libwebp
+  run:
+    - python >3.5
+    - numpy >=1.18.5
+    - pandas
+    - rasterio
+    - click >=7
+    - tqdm
+    - geopandas
+    - Pillow
```

Comment:
package names should be lowercase

Reply:
I tested this and it works. The change is made.